tribits/ci_support/CDashQueryAnalyzeReport.py

```diff
@@ -2193,6 +2410,415 @@ def createCDashTestHtmlTableStr(
     htmlStyle=htmlStyle, htmlTableStyle=htmlTableStyle )
 
 
+def getCDashTestHtmlTableConsecColData(testsetTableType):
+  tcd = TableColumnData
+  if testsetTableType == 'nopass':
+    consecCol = tcd("Consec&shy;utive Non-pass Days", 'consec_nopass_days', 'right')
+  elif testsetTableType == 'pass':
+    consecCol = tcd("Consec&shy;utive Pass Days", 'consec_pass_days', 'right')
+  elif testsetTableType == 'missing':
+    consecCol = tcd("Consec&shy;utive Missing Days", 'consec_missing_days', 'right')
+  else:
+    raise Exception("Error, invalid testsetTableType="+str(testsetTableType))
+  return consecCol
+
+
+# Class to generate the data for an HTML report for all test-sets for a given
+# issue tracker.
+#
+# This class can be reused for multiple issue issue trackers.
+#
+class IssueTrackerTestsStatusReporter(object):
+
+  # Constructor
+  #
+  # cdashReportData [persisting]: Data used to create the final report (of type
+  # CDashReportData).
+  #
+  def __init__(self, verbose=True):
+    self.cdashReportData = CDashReportData()
+    self.testsetsReporter = TestsetsReporter(
+      self.cdashReportData, testsetAcroList = ['twip', 'twim', 'twif', 'twinr'],
```

Comment:
Remove override of testsetAcroList and let it operate on all test sets passed to it.

Reply:
Done